common/scripted_triggers/wc_scripted_triggers.txt

```diff
@@ -4971,4 +4983,7 @@ this_can_lend_prev_money_trigger = {
 	trigger_else = {
 		opinion = { who = PREV value = -20 }
 	}
+}
+can_get_mutation_healing_trigger = {
+	check_variable = { which = void_mutations value = 4 }
```

Comment:
So, 4 mutation is max?

Reply:
yeah i wanted to slow down the process of becoming a nraqi with this

Reply:
But why 5 is max here?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81498705-88dc8a00-92d7-11ea-8c51-72547ca8dba0.png)

</details>

Reply:
cause i wanted to limit amount of mutations you can get through healing so its an artifical limit while the one in mutation event checks for max amount of traits your char can have